Troubleshooting for M1 errors caused by Rosetta (#5687)
cyberbotics/webots: docs/guide/installation-procedure.md

Alternatively, you can double-click on the Webots icon to launch it.

> #### Trouble Shooting for Mac M1 Users

Reviewer: I wouldn't mention M1 only as M2 processors are likely to show up the same problem.

```suggestion
#### Troubleshooting for Apple Silicon Users
```

Author: Sounds good. Thank you.

If you are geting errors like these
`...(mach-o file, but is an incompatible architecture (have (arm64), need (x86_64)))`

> This is caused by Rosetta loading Webots under the x86 architecture instead of the native ARM. Unless if you plan on using x86 binaries with your Webots simulator, make sure to turn off Rosetta. Sometimes it may try to open the app using Rosetta by default, which may cause issues with running your code inside the simulator. 

Reviewer: ```suggestion
This is likely caused by Rosetta loading Webots under the x86 architecture instead of the native ARM. Unless you really want to use x86 binaries with your Webots simulation, make sure to turn off Rosetta. Sometimes macOS may try to open the app using Rosetta by default, which may cause issues when it comes to run robot controllers with dependencies on ARM libraries.
```

> To check if it's opened using Rosetta, right click on the Webots application in Finder and select Get Info. 

Reviewer: ```suggestion
To check if it's opened using Rosetta, right click on the Webots application in Finder and select "Get Info". 
```

> Make sure open using Rosetta setting is like the picture below. 

Reviewer: ```suggestion
Make sure that the "Open using Rosetta" setting is unchecked, like in the picture below: 
```

> ![Screenshot 2022-12-21 at 9 55 39 AM](https://user-images.githubusercontent.com/33169711/208961163-6a0826da-0247-4e9a-91e8-6e053cc29d5e.png)

Reviewer: ```suggestion
%figure "Rosetta setting"

![rosetta_setting.png](images/rosetta_setting.png)

%end
```

Alternatively, you can double-click on the Webots icon to launch it.

#### Troubleshooting for Apple Silicon Users

> If you are geting errors like these

Reviewer: Typo and test failure:
```suggestion
If you are getting errors like these:
```
